packages/api/tests/endpoints/test-granules-get.js

```diff
@@ -543,39 +497,7 @@ test.serial('GET returns a 404 response if the granule is not found', async (t)
   t.is(message, 'Granule not found');
 });
 
-// TODO postgres query doesn't return searchContext
```

Comment:
Is there any value in testing pagination here? Probably not as it's verified elsewhere and not really specific to granules. Just thinking out loud.

Reply:
I think... that's not necessary. It seems like it's all covered, it just took some time to figure out _where_ everything is covered.